codemagic.yaml

```diff
@@ -0,0 +1,13 @@
+workflows:
+    production:
+        name: Deploy to production
+        environment:
+            groups:
+                - aws-production
+                - production
+        scripts:
+            - name: Deploy to production
+              script: |
+                  set -e
+                  aws s3 sync ./your-first-flutter-app-to-appstore s3://labs.codemagic.io --delete --cache-control max-age=31536000,public
```

Comment:
I believe no need for `--cache-control max-age=31536000,public`

Reply:
it's needed